lib/msal-browser/apiReview/msal-browser.api.md

```diff
@@ -623,6 +623,7 @@ export const EventType: {
     readonly LOGIN_FAILURE: "msal:loginFailure";
     readonly ACQUIRE_TOKEN_START: "msal:acquireTokenStart";
     readonly ACQUIRE_TOKEN_SUCCESS: "msal:acquireTokenSuccess";
+    readonly BROKERED_REQUEST_SUCCESS: "msal:brokeredRequestSuccess";
```

Comment:
Do we need a BROKERED_REQUEST_START/FAILURE to go along with this?

Reply:
That'll happen in a PR with the other event type changes

Reply:
Got it, sounds good thanks!